<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFaultDataTable</class>
 <widget class="QDialog" name="CFaultDataTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>故障数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="listFaultData">
        <column>
         <property name="text">
          <string>点逻辑名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>故障类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>初始值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>目标值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>正常值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>变化时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>变化方式</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>延迟时间</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonModify">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDel">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CFaultDataTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
